core/components/DiscordBot/commands/status.ts

```diff
@@ -202,7 +202,7 @@ export const removeOldEmbed = async (interaction: ChatInputCommandInteraction, t
     const oldMessageId = txAdmin.persistentCache.get('discord:status:messageId');
     if (typeof oldChannelId === 'string' && typeof oldMessageId === 'string') {
         const oldChannel = await interaction.client.channels.fetch(oldChannelId);
-        if (oldChannel?.type === ChannelType.GuildText) {
+        if (oldChannel?.type === ChannelType.GuildText || oldChannel?.type === ChannelType.GuildAnnouncement) {
             await oldChannel.messages.delete(oldMessageId);
         } else {
             throw new Error(`oldChannel is not a guild text channel`);
```

Comment:
please highlight the fact that its not an announcement channel either when you throw (like you did with every other error)

Reply:
Ah missed that